.github/workflows/flutter-action-build-and-integration-testing.yml

```diff
@@ -0,0 +1,57 @@
+name: Flutter actions build and integration test
+
+on:
+  push:
+    branches:
+      - 'master'
+    paths-ignore:
+      - 'documentation/**'
+      - 'LICENSE'
+      - 'README.md'
+      - '.gitignore'
+      - 'pre-commit-config.yaml'
+
+defaults:
+  run:
+    working-directory: ./app
+
+jobs:
+  build_and_integration_test:
+    strategy:
+      matrix:
+        device:
+          - iPhone 12 Pro Max (14.2)
+      fail-fast: false
+    runs-on: macOS-latest
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v2
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: '12.x'
+      - name: Set up Flutter actions
+        uses: subosito/flutter-action@v1
+        with:
+          flutter-version: '1.22.5'
+          channel: 'stable'
+      - name: List all available devices
```

Comment:
Grabbing devices as per [this example](https://medium.com/flutter-community/run-flutter-driver-tests-on-github-actions-13c639c7e4ab)